modules/rest-api/pages/rest-get-memory-information.adoc

```diff
@@ -0,0 +1,100 @@
+= Getting Memory Information
+:description: pass:q[Use REST API to retrieve the current memory quota information for services in a cluster.]
+:page-toclevels: 2
+
+[abstract]
+{description}
+
+[#http-method-and-uri]
+== HTTP Method and URI
+
+----
+GET /pools/default
+----
+
+[#description]
+== Description
+
+Allows to view memory quotas for the Multi-Dimensional Scaling (MDS) services Index, Search, Eventing, and Analytics.
+Query and Backup services do not have a memory allocation.
+If no custom quota was specified during configuration for one or more services, those services retain the default allocations.
+These are the default allocations:
+
+* 256 Mb for fts (Search Service)
+* 512 Mb for index (Index Service)
+* 256 Mb for eventing (Eventing Service)
+* 1024 Mb for cbas (Analytics Service)
+
+For information about the maximum memory allocation permitted for a node, see xref:learn:buckets-memory-and-storage/memory.adoc#service-memory-quotas[Service Memory Quotas].
+
+== curl Syntax
+
+----
+curl -v -X GET <network_address>:<port_number>/pools/default \
+ -u <username>:<password>
+----
+
+== Required Permissions
+
+`Full Admin` role.
+
+[#responses]
+== Responses
+
+|===
+|Value | Description  
+
+| `200 OK` and JSON array containing the expired backups and their details.
+| Successful call.
+
+| `400`
+| Invalid parameter.
+
+| `400 Object Not found`
+| The repository in the endpoint URI does not exist.
+
+| `401 Unauthorized`
+|  Authorization failure due to incorrect username or password.
+
+| `403 Forbidden`, plus a JSON message explaining the minimum permissions.
+| The provided username has insufficient privileges to call this method.
+
+| `404 Object Not Found`
+| Error in the URI path.
+
+| `500 Internal Server Error`
+| Error in Couchbase Server.
+|===
+
+[#example]
+== Example
+
+The following example retrieves memory information for each service.
+
+----
+curl -v -X GET http://127.0.0.1:8091/pools/default -u Administrator:password | jq '.'
+----
+
+If the call is successful, the response is similar to the following.
+
+The output of the call is piped to https://stedolan.github.io/jq/[jq^] to improve readability.
+
+----
+include::partial$get_memory_information_sample.json[]
+----
+
+In the example output, the services memory quotas are listed as follows in these parameters:
+  
+* `queryMemoryQuota` for the Query Service.
```

Comment:
* `memoryQuota` for the Data Service.
* `queryMemoryQuota` for the Query Service.